### PR TITLE
Add more information to items.

### LINKED ISSUE #851: Add copy_number in the get_items_for_bib method
child of #751
The 
​`/almaws​/v1​/bibs​/{mms_id}​/holdings​/{holding_id}​/items`
Retrieve Items list https://developers.exlibrisgroup.com/console/?url=/wp-content/uploads/alma/openapi/bibs.json#/Catalog/get%2Falmaws%2Fv1%2Fbibs%2F%7Bmms_id%7D%2Fholdings%2F%7Bholding_id%7D%2Fitems provides 
the copy_id: 
copy_id | string | The item's copy ID, used to identify individual copies of the same title when multiple copies exist at the same location.
-- | -- | --

https://bibdata.princeton.edu/bibliographic/426420/items

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #851

### CODE REVIEW

File: app/adapters/alma_adapter.rb [hunk 1]
```diff
@@ -67,10 +67,38 @@ def format_holding(holding_items_array)
...
+      "items" => holding_items_array.each_with_index.map { |bib_item, idx| format_item(bib_item, idx) }
     }
   end
 
+  def format_item(bib_item, idx)
+    bib_item.item["item_data"].merge(
+      "id" => bib_item.item["item_data"]["pid"],
+      "copy_number" => bib_item.item["holding_data"]["copy_id"].to_i,
+      "item_sequence_number" => idx + 1,
+      "temp_location" => temp_location(bib_item),
```
Comment (christinach) [inline app/adapters/alma_adapter.rb:79]: I think we should remove the item_sequence_number based on https://github.com/pulibrary/marc_liberation/issues/845#issuecomment-709450744. @mzelesky can you check?

**Review by christinach — CHANGES_REQUESTED** [review]

File: app/adapters/alma_adapter.rb (hunk 1 above)
Comment (mzelesky) [reply app/adapters/alma_adapter.rb:79]: There is no item sequence number in Alma. Until we have a full dump, I won't know if the sequence of items returned in the publishing job is significant (meaning that the order is intentional). We may need to sort items by enumeration.

**Comment by tpendragon** [comment]
@christinach Sure. Removed.

**Review by christinach — APPROVED** [review]

**Comment by christinach** [comment]
@tpendragon a rubocop error?

**Comment by tpendragon** [comment]
> @tpendragon a rubocop error?

Fixed.

**Pull request merged**